### PR TITLE
Sound tweaks: Suit storage, floors.

### PR DESCRIPTION
**Pull request by PetMudstone**
### Purpose
- The suit storage unit uses the locker's closing and opening sounds.
- Added various floor sounds groups and made various floors use them.
- Assigned sounds to the various "Step Replacer" components found on some items.

### Notes
- The floor sounds and step replacer don't actually seem to work at the moment.

### CODE REVIEW

**Comment by corp-0** [comment]
I tested this on my local but since I'm part of the contributors, I feel the moral thing to do would have someone else test and merge.

File: UnityProject/Assets/Scripts/Clothing/StepChanger.cs [hunk 1]
```diff
@@ -1,115 +1,93 @@
...
+		/// <summary>
+		/// Stupid logic to handle all possible interaction combinations with clownshoes and hardsuits
+		/// </summary>
+		private void HandleTakingOff()
+		{
+			switch (hasPriority)
 			{
-				Player = info.ToPlayer;
-				mind.StepSound = SoundChange;
+				case true when mind.SecondaryStepSound:
```
Comment (NoooneyDude) [inline UnityProject/Assets/Scripts/Clothing/StepChanger.cs:74]: C sharp is fascinating.

**Comment by NoooneyDude** [comment]
This looks good to me. What is the intent of the cherry-pick label, curiously?

**Comment by corp-0** [comment]
I’m tagging prs that need to be cherry picked/back ported to 4119. I intend to give a lot of love to 4120.

**Comment by NoooneyDude** [comment]
Aah. Smart.

**Pull request merged**